rfcs/input-component.md

```diff
@@ -125,12 +123,6 @@ export default () => {
   Defines styles for inputs that are grouped with other controls.
 * `size: 'default' | 'compact'`
   Defines the size of an input control.
-* `onChange: Function` - Optional
-  onChange event handler.
-* `onFocus: Function` - Optional
-  onFocus event handler.
-* `onBlur: Function` - Optional
```

Comment:
The `onFocus` and `onBlur` are overwritten to handle internal `isFocused` state.